.circleci/config.yml

```diff
@@ -0,0 +1,34 @@
+version: 2.1
+
+orbs:
+  composer: itinerisltd/composer@0.2
+
+workflows:
+  test:
+    jobs:
+      - composer/install:
+          name: test
+          executor: composer/seven
+          pre-steps:
+            - run: sudo docker-php-ext-enable xdebug
+            - run: curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ~/cc-test-reporter
```

Comment:
:+1: downloading the test reporter in ci is better than baking this in the docker image